packages/compass-e2e-tests/tests/instance-my-queries-tab.test.ts

```diff
@@ -114,13 +114,14 @@ describe('Instance my queries tab', function () {
     const renameModal = await browser.$(Selectors.RenameSavedItemModal);
     await renameModal.waitForDisplayed();
 
-    await browser
-      .$(Selectors.RenameSavedItemModalTextInput)
-      .setValue(newFavoriteQueryName);
+    await browser.setValueVisible(
+      Selectors.RenameSavedItemModalTextInput,
+      newFavoriteQueryName
+    );
     const confirmRenameButton = await browser.$(
       Selectors.RenameSavedItemModalSubmit
     );
-    confirmRenameButton.waitForEnabled();
+    await confirmRenameButton.waitForEnabled();
```

Comment:
Again - a bunch of promises we forgot to await which probably explains why the debug screenshots exist :)